docs/source/pillars/information_governance.md

```diff
@@ -177,6 +177,9 @@ _The ability of the TRE organisation to ensure that people with access to data a
 * - You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data
   - This may include multi-factor authentication (MFA), ID checks or email/phone verification
   - Mandatory
+* - You have a clear method of communicating the level of security controls present in the TRE for all roles within your TRE organisation
```

Comment:
```suggestion
* - You must clearly communicate the security controls of your TRE to all roles.
```

I've tried to tidy that but I'm not exactly sure what it means or how it relates to other additions in this PR.

Again, I'm not sure if this is really about the controls.

Reply:
I think this can be incorporated with the above about communicating what your TRE can/can't do

Reply:
OK, feel like the consensus is to remove this?

Reply:
Updated in https://github.com/sa-tre/satre-specification/commit/eefc4621c858d9c429ceac51f1f52963563b6528